src/Servers/HttpSys/src/RequestProcessing/Response.cs

```diff
@@ -31,6 +31,9 @@ internal sealed class Response
     private HttpApiTypes.HTTP_RESPONSE_V2 _nativeResponse;
     private HeaderCollection? _trailers;
 
+    private const string RespectHttp10KeepAliveSwitch = "Microsoft.AspNetCore.Server.HttpSys.RespectHttp10KeepAlive";
```

Comment:
I guess we don't need a follow up work item since it's already "removed" in 9.0.